ooiservices/migrations/versions/4ddc43e8e5b3_testing_migrations.py

```diff
@@ -1,352 +0,0 @@
-    sa.Column('is_recovered', sa.Boolean(), server_default='false', nullable=False),
-    sa.ForeignKeyConstraint(['deployment_id'], [u'ooiui_testing.deployments.id'], ),
-    sa.ForeignKeyConstraint(['stream_id'], [u'ooiui_testing.streams.id'], ),
-    sa.PrimaryKeyConstraint('id'),
-    schema='ooiui_testing'
-    )
-    op.create_table('stream_parameter_link',
-    sa.Column('id', sa.Integer(), nullable=False),
-    sa.Column('stream_id', sa.Integer(), nullable=False),
-    sa.Column('parameter_id', sa.Integer(), nullable=False),
-    sa.ForeignKeyConstraint(['parameter_id'], [u'ooiui_testing.stream_parameters.id'], ),
-    sa.ForeignKeyConstraint(['stream_id'], [u'ooiui_testing.streams.id'], ),
-    sa.PrimaryKeyConstraint('id'),
-    schema='ooiui_testing'
-    )
-    op.create_table('dataset_keywords',
-    sa.Column('id', sa.Integer(), nullable=False),
-    sa.Column('dataset_id', sa.Integer(), nullable=False),
-    sa.Column('concept_name', sa.Text(), nullable=True),
-    sa.Column('concept_description', sa.Text(), nullable=True),
-    sa.ForeignKeyConstraint(['dataset_id'], [u'ooiui_testing.datasets.id'], ),
-    sa.PrimaryKeyConstraint('id'),
-    schema='ooiui_testing'
-    )
-    ### end Alembic commands ###
-
-
-def downgrade():
-    ### commands auto generated by Alembic - please adjust! ###
-    op.drop_table('dataset_keywords', schema='ooiui_testing')
-    op.drop_table('stream_parameter_link', schema='ooiui_testing')
-    op.drop_table('datasets', schema='ooiui_testing')
-    op.drop_table('instrument_deployments', schema='ooiui_testing')
-    op.drop_table('driver_stream_link', schema='ooiui_testing')
-    op.drop_table('drivers', schema='ooiui_testing')
-    op.drop_table('streams', schema='ooiui_testing')
-    op.drop_table('platform_deployments', schema='ooiui_testing')
-    op.drop_table('annotations', schema='ooiui_testing')
-    op.drop_table('inspection_status', schema='ooiui_testing')
-    op.drop_table('platforms', schema='ooiui_testing')
-    op.drop_table('asset_file_link', schema='ooiui_testing')
-    op.drop_table('user_scope_link', schema='ooiui_testing')
-    op.drop_table('instruments', schema='ooiui_testing')
-    op.drop_table('installation_records', schema='ooiui_testing')
-    op.drop_table('users', schema='ooiui_testing')
-    op.drop_table('assets', schema='ooiui_testing')
-    op.drop_table('instrument_models', schema='ooiui_testing')
-    op.drop_table('files', schema='ooiui_testing')
-    op.drop_table('organizations', schema='ooiui_testing')
-    op.drop_table('deployments', schema='ooiui_testing')
-    op.drop_table('stream_parameters', schema='ooiui_testing')
-    op.drop_table('assemblies', schema='ooiui_testing')
-    op.drop_table('arrays', schema='ooiui_testing')
-    op.drop_table('manufacturers', schema='ooiui_testing')
-    op.drop_table('asset_types', schema='ooiui_testing')
-    op.drop_table('user_scopes', schema='ooiui_testing')
-    ### end Alembic commands ###
```

Comment:
I have a feeling that our migrations should really be mirroring the RDS instance. @oceanzus what do you think?